### DISCUSSION TITLE
Can't to update quantity value data type fields through swagger API

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#15694] [Q&A / Support]
@kingjia90

Hello Jia Jia Ji, we have generated the rest API in pimcore, to update a product from ERP to Pimcore, the input data type fields can be able to update, can't to update the Quantity Value data type fields, the integer value only can be able to update can't to update the unit values. I wanted to update both value and unit through swagger API of the Pimcore. Can you please help if you have any knowledge about this.

Thank You!

**Comment by kingjia90** [comment]
Do you mean Data Hub Simple Rest? Can you please share any screenshot or snippets to see what is the current state? I've found out some problems with the Data Importer https://github.com/pimcore/data-importer/pull/345#issuecomment-1651439537, maybe some similar fixes/adaptations are necessary for the Simple Rest too.

**Comment by KasthuriAnandhan** [comment]
Yes I meant Data Hub Simple Rest. But I'm asking can we able to include unit value in swagger API like mentioned below.

![Screenshot 2023-08-07 124813](https://github.com/pimcore/pimcore/assets/125338676/9b8a3cce-b943-49e1-bb49-c523b2d87500)

the above values Height, Width, length, Cubage and weight have the unit values how to update those unit values from here.

**Comment by KasthuriAnandhan** [comment]
I do know is my request is possible to achieve and pimcore have any inbuilt option for this, if anything is there like that let me know please.
